delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/wiring/DeltaModule.scala

```diff
@@ -63,13 +64,13 @@ class DeltaModule(appCfg: AppConfig, config: Config)(implicit classLoader: Class
       statisticsCtx <- ioJsonContentOf("contexts/statistics.json") // TODO: Should be moved to views?
     } yield RemoteContextResolution
       .fixed(
-        contexts.error      -> errorCtx,
-        contexts.metadata   -> metadataCtx,
-        contexts.search     -> searchCtx,
-        contexts.tags       -> tagsCtx,
-        contexts.version    -> versionCtx,
-        contexts.offset     -> offsetCtx,
-        contexts.statistics -> statisticsCtx
+        contexts.error      -> errorCtx.topContextValueOrEmpty,
```

Comment:
To avoid to repeat this `.topContextValueOrEmpty`, we could have a loadContext method ?
And for our static contexts, the method should fail if no context can be loaded.